homeassistant/components/seventeentrack/sensor.py

```diff
@@ -95,14 +86,32 @@ async def async_setup_platform(
         _LOGGER.error("There was an error while logging in: %s", err)
         return
 
-    scan_interval = config.get(CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL)
+    hass.async_create_task(
+        hass.config_entries.flow.async_init(
+            DOMAIN, context={"source": SOURCE_IMPORT}, data=config
+        )
+    )
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up a 17Track sensor entry."""
+
+    entry_id = config_entry.entry_id
+
+    client = hass.data[DOMAIN][entry_id]
+
+    scan_interval = config_entry.options.get(CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL)
```

Comment:
Currently this is never set or imported afaict

Reply:
see Line 314:
`self.async_update = Throttle(self._scan_interval)(self._async_update)`

Reply:
yes, but I am talking about how this data got into `config_entry.options`